tests/rptest/tests/rpk_config_test.py

```diff
@@ -0,0 +1,183 @@
+# Copyright 2021 Vectorized, Inc.
+#
+# Use of this software is governed by the Business Source License
+# included in the file licenses/BSL.md
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0
+
+from ducktape.utils.util import wait_until
+
+from rptest.tests.redpanda_test import RedpandaTest
+from rptest.clients.rpk_remote import RpkRemoteTool
+from rptest.services.rpk_consumer import RpkConsumer
+
+import yaml
+
+import random
+
+
+class RpkConfigTest(RedpandaTest):
+    def __init__(self, ctx):
+        super(RpkConfigTest, self).__init__(test_context=ctx)
+        self._ctx = ctx
+
+    def test_config_init(self):
+        n = random.randint(0, len(self.redpanda.nodes))
+        node = self.redpanda.get_node(n)
+        rpk = RpkRemoteTool(self.redpanda, node)
+        path = './redpanda-test.yaml'
+        rpk_lines = rpk.config_init(path)
+        rpk_out = '\n'.join(rpk_lines)
+        rpk_expected_out = "Couldn't find config file at ./redpanda-test.yaml. Generating it.\n"
+        assert rpk_out == rpk_expected_out
+
+        cat_lines = node.account.ssh_capture(f'cat {path}')
```

Comment:
I think here you can use `node.account.file_copy` or something like that I can't remember the exact method name

Reply:
I was looking at the docs and it seems you can only copy stuff between nodes: https://ducktape.readthedocs.io/_/downloads/en/latest/pdf/ (copy_between).

I did find a simpler function, `ssh_output` which makes the joining unnecessary.

Reply:
@0x5d  how about? `node.account.copy_from(RedpandaService.DATA_DIR, d)` looks like what you're after

Reply:
@0x5d how about? node.account.copy_from(RedpandaService.DATA_DIR, d) looks like what you're after

Reply:
I had somehow missed that one. Thanks!